pydicom/tests/test_gdcm_pixel_data.py

```diff
@@ -63,18 +71,27 @@
 save_dir = os.getcwd()
 
 
-@pytest.mark.skipif(have_gdcm_handler, reason=gdcm_present_message)
 class GDCM_JPEG_LS_Tests_no_gdcm(unittest.TestCase):
     def setUp(self):
-        self.jpeg_ls_lossless = read_file(jpeg_ls_lossless_name)
+        self.utf8_filename = os.path.join(tempfile.gettempdir(), "ДИКОМ.dcm")
+        shutil.copyfile(jpeg_ls_lossless_name, self.utf8_filename)
+        try:
+            self.jpeg_ls_lossless = read_file(
+                unicode(self.utf8_filename,
+                        encoding=sys.getfilesystemencoding()))
+        except NameError:
```

Comment:
I think you have to use utf-8 encoding here, as the file system encoding may be different (usually under Windows),